tests/lvms/test_lvms_vllm_on_intel_hpu.sh

```diff
@@ -18,9 +18,8 @@ function build_docker_images() {
     cd $WORKPATH
     echo $(pwd)
 
-    git clone https://github.com/HabanaAI/vllm-fork.git
-    cd ./vllm-fork/
-    git checkout f78aeb9da0712561163eddd353e3b6097cd69bac # revert this to habana_main when https://github.com/HabanaAI/vllm-fork/issues/1015 is fixed
+    VLLM_FORK_VER=f78aeb9da0712561163eddd353e3b6097cd69bac # revert this to habana_main when https://github.com/HabanaAI/vllm-fork/issues/1015 is fixed
+    git clone -b ${VLLM_FORK_VER} --single-branch https://github.com/HabanaAI/vllm-fork.git && cd ./vllm-fork
```

Comment:
The linked issue was marked as fixed already ~3 months ago, and you cannot use `-b` option with individual commits, only with branch and tag names.  Please fix.

Reply:
fix in [f12616c](https://github.com/opea-project/GenAIComps/pull/1790/commits/f12616ca3b503a52a3522d9042317c6d88839c78)